civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -310,4 +311,21 @@
     $('#' + this.id + '-' + 'day').val(dateValues.date).trigger('change');
     $('#' + this.id + '-' + 'year').val(dateValues.year).trigger('change');
   }
+
+  /**
+   * Do the stuff related to Users List Page
+   */
+  function usersListPage () {
+    var isUsersListPage = $('.page-users-list').length;
```

Comment:
Can we do one of two things here:

1. rename this function to `onUsersListPage`, or
2. refactor whatever calls this function to this:
```js
$('.page-users-list').length && usersListPage();
```
And remove this condition from here.

I'm suggesting this so it's more clear that these changes only occur for the users list page and we are not implying that they would run for every page.

Reply:
As discussed, it is not implemented.